### ISSUE TITLE
Receive sound not working in Linux snap

### ISSUE THREAD

**Issue by dr3wbr0** [BlueBubblesApp/bluebubbles-app#2692]
BB version: 1.12.7.0_Linux_Snap
OS: Ubuntu 22.04.4 LTS
Kernel: Linux 6.1.0-1034-oem x86_64 GNU/Linux

Settings -> Conversation Settings -> Sounds -> Change Receive Sound

Send sound is working but receive sound doesn't work in chat view or with BB minimized. I set an mp3 for both sounds.

**Comment by jurassicjordan** [comment]
I'm having this problem for both send and receive. no sound at all

BB ver: 1.12.7.0_Linux_Flatpak 
OS: EndeavourOS Linux x86_64
Kernel: 6.9.5-zen1-1-zen

**Comment by jjoelj** [comment]
Is this still an issue?

**Comment by dr3wbr0** [comment]
No, this is from a few years ago. Newer releases fixed it for me, thanks.
